rfc/rfc002-authentication-token.md

```diff
@@ -401,3 +401,129 @@ Beside the mandatory VP fields, the following applies:
 * The `proof.type` field MUST equal `NutsUziSignedContract`.
 * The `proof.proofValue` field MUST contain the JWT in its compact serialization form as described in \[[RFC7515 section 3.1](https://tools.ietf.org/html/rfc7515#section-3.1)\].
 
+### 7.3 NutsEmployeeIdentity
```

Comment:
UZI and IRMA means are called just so, `NutsEmployeeIdentity` is the name of the credential. I think the means should be called something along the line of "Self-claimed Employee Identity" or just "Employee Identity".


Also; should be now add levels of assurances to the methods we spec'd? (or refer to a framework we use, e.g. eIDAS(2)?)